lib/octobox_notifier/system_notification.rb

```diff
@@ -53,8 +53,8 @@ def pluralize(n, str)
         "#{n} #{str}#{'s' unless n == 1}"
       end
 
-      def notification_changed
-        current_ids != previous_ids
+      def new_notifications?
+        !(current_ids - previous_ids).empty?
```

Comment:
Prefer to avoid `!`: `(current_ids - previous_ids).any?`

Reply:
TIL